content/telegraf/v1.24/configure_plugins/custom_builder.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Reduce Telegraf binary size with customized agent
+description: Use the custom builder tool to reduce Telegraf binary disk space. 
+menu:
+  telegraf_1_24:
+
+    name: Reduce binary size
+    Parent: Configure plugins
+    weight: 20
+---
+Reduce Telegraf's overall memory and disk footprint with the custom builder. This tool allows you to select which plugins to use in the Telegraf binary based on your configuration file. 
+
+## Requirements
+
+- Ensure you've installed the current version of [Go](https://go.dev/). 
+- Create your Telegraf configuration file with the plugins you want to use. For details, see [Configuration options](/telegraf/v1.24/configuration/).
```

Comment:
This seems pretty vague. If anything, "current" should be "latest stable." Is there a minimum supported version of go?

```suggestion
- Latest stable version of [Go](https://go.dev/). 
- Telegraf configuration file with the plugins you want to use. For details, see [Configuration options](/telegraf/v1.24/configuration/).
```

Reply:
in my blog i have " At the time of publishing, this tool requires Go version 1.18.0+"